src/extension/content-script/batteries/Medium.ts

```diff
@@ -4,65 +4,30 @@ import { findLightningAddressInText, setLightningData } from "./helpers";
 const urlMatcher = /^https:\/\/(.+\.)?medium.com\/(\S+)?/;
 
 const battery = (): void => {
-  const authorLinkSelector = getAuthorLinkSelector();
-  if (!authorLinkSelector) return;
-
-  const shortBio = document.querySelector<HTMLParagraphElement>(
-    `${authorLinkSelector}~p:last-of-type`
-  )?.innerText;
-  if (!shortBio) return;
-
-  const match = findLightningAddressInText(shortBio);
+  const name =
+    document.querySelector<HTMLHeadingElement>(".pw-author-name")?.innerText;
```

Comment:
can we be sure that there is no other element (like a comment) that has that class?

Reply:
medium.com doesn't seem to use semantics at all and barely uses meaningful class names...
Also, this is only for the profile page so there shouldn't be any comments (and hence authors of any other kind) here as it's a showcase of the author's posts only.

Reply:
ok, thanks.